tests/test_features/test_interview/test_question_generator.py

```
@@ -39,7 +39,7 @@ def test_first_turn_question_generation(first_turn_state, monkeypatch):
     monkeypatch.setattr(
         question_generator,
         "get_llm",
-        lambda temperature=0.7: _mock_llm_return("첫 질문입니다."),
+        lambda model=None, temperature=0.7: _mock_llm_return("첫 질문입니다."),
```

Ruff (0.15.2)

[warning] Unused lambda argument: `model`

(ARG005)